Formula/w/weaviate.rb

```diff
@@ -0,0 +1,30 @@
+class Weaviate < Formula
+  desc "Open-source vector database that stores both objects and vectors"
+  homepage "https://weaviate.io/developers/weaviate/"
+  url "https://github.com/weaviate/weaviate/archive/refs/tags/v1.27.6.tar.gz"
+  sha256 "d8cad0339ccfd081be6afe1afd879f64bba224ed7ecbdb8b974853b536584330"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s
+      -w
```

Comment:
```suggestion
      -s -w
```

Reply:
IIRC, I tried this syntax, but it resulted in a brew audit warning, do you think it's the preferred option @daeho-ro ?

Reply:
mea culpa, seems to be working. I might have got the wrong line number the first time.
Should be fixed now